src/workflow/CompareCommitToRevisionWorkflow.php

```diff
@@ -51,7 +51,7 @@ public function getCommandHelp() {
   }
 
   public function requiresWorkingCopy() {
-    return true;
+    return $this->getArgument('use-local-enlistment');
```

Comment:
Is this just checking for the existence of the `use-local-enlistment` flag, or is it also checking for truthiness?

If it's just checking for existence, would we need this instead:

```
return $this->getArgument('use-local-enlistment') === true
```

?

Reply:
It's a "switch" param, determined by how it is defined:  https://github.com/uber/arcanist/blob/stable/src/workflow/CompareCommitToRevisionWorkflow.php#L104-L108. 

If it has a `param` option, then arc expects it to have a param.  

so, `--use-local-enlistment false` or `--use-local-enlistment=false` both error out as invalid CLI params.